dev/io.openliberty.io.netty/bnd.bnd

```diff
@@ -0,0 +1,63 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+nettyVersion=4.1.59.Final
+
+Bundle-Name: Netty as an OSGi bundle
+Bundle-SymbolicName: io.openliberty.io.netty
+Bundle-Description: Netty as an OSGi bundle for OpenLiberty, version ${bVersion}
+
+Import-Package: \
+  !com.google.protobuf.nano,\
+  !com.google.code.gson,\
+  !org.checkerframework,\
+  !com.ibm.websphere.monitor.jmx,\
```

Comment:
This package shouldn't need to be excluded (monitor isn't on the buildpath)

Reply:
Agreed, I will check both the bnd files.

Reply:
Done